shell/components/dialog/ForcePodRemoveDialog.vue

```diff
@@ -0,0 +1,218 @@
+<script>
+import { alternateLabel } from '@shell/utils/platform';
+import AsyncButton from '@shell/components/AsyncButton';
+import { escapeHtml } from '@shell/utils/string';
+import { Banner } from '@components/Banner';
+import { Card } from '@components/Card';
+import Checkbox from '~/pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue';
+import { uniq } from '@shell/utils/array';
+
+export default {
+  name: 'ForcePodRemoveDialog',
+
+  components: {
+    AsyncButton,
+    Banner,
+    Card,
+    Checkbox
+  },
+
+  props:      {
+    resources: {
+      type:     Array,
+      required: true
+    }
+  },
+
+  data() {
+    return {
+      errors:      [],
+      confirmName: '',
+      forceDelete: true
+    };
+  },
+
+  computed: {
+    names() {
+      return this.resources.map(obj => obj.metadata.name);
+    },
+
+    machine() {
```

Comment:
`machine`, `nameToMatch`, `type` and `forceRemoveMachine` should be removed because they are related to machines and that shouldn't be necessary for deleting workoads